controller/controller_server/csi_controller_server.py

```diff
@@ -47,10 +48,8 @@ def CreateVolume(self, request, context):
         try:
             utils.validate_create_volume_request(request)
         except ObjectIdError as ex:
-            logger.exception(ex)
-            context.set_details(ex.message)
-            context.set_code(grpc.StatusCode.NOT_FOUND)
-            return csi_pb2.CreateVolumeResponse()
+            return handle_exception(ex, context, grpc.StatusCode.NOT_FOUND,
+                                    csi_pb2.CreateVolumeResponse)
```

Comment:
I don't see how this is related to the PR

Reply:
It came with `git pull -r`